### PR TITLE
fix/forbidden-path-check-ns-files

### PR DESCRIPTION
**Pull request by brian-mulier-p**
prevent any update or retrieval on given forbidden paths (to prevent the ability for eg. of pushing a "_flows" directory resulting in breaking the UI as it's a simulated folder)

### CODE REVIEW

File: webserver/src/main/java/io/kestra/webserver/controllers/NamespaceFileController.java [hunk 1]
```diff
@@ -49,6 +49,10 @@ public class NamespaceFileController {
         }
     }
 
+    private final List<Pattern> forbiddenPathPatterns = List.of(
+        Pattern.compile("/_flows.*")
```
Comment (loicmathieu) [inline webserver/src/main/java/io/kestra/webserver/controllers/NamespaceFileController.java:53]:
```suggestion
        Pattern.compile("^/_flows.*")
```

I think it's more correct to add `^` to denote the beginning of the line

**Review by loicmathieu — CHANGES_REQUESTED** [review]

File: webserver/src/main/java/io/kestra/webserver/controllers/NamespaceFileController.java (hunk 1 above)
Comment (brian-mulier-p) [reply webserver/src/main/java/io/kestra/webserver/controllers/NamespaceFileController.java:53]: ups forgot, you're right

Comment (brian-mulier-p) [reply webserver/src/main/java/io/kestra/webserver/controllers/NamespaceFileController.java:53]: nvm as I use "matches" method, it automatically prepend with "^" and append with "$" so it's already an entire match regex

**Review by loicmathieu — APPROVED** [review]
LGTM

**Pull request merged**